include/glow/Backend/BackendUtils.h

```diff
@@ -102,7 +102,15 @@ class RuntimeBundle {
   static runtime::RuntimeBundle create(const IRFunction &F,
```

Comment:
Consider updating the comment for this function to include the new parameter `contiguous`.